### DISCUSSION TITLE
How do I delete cases after creating a test case?

### DISCUSSION THREAD

**Discussion by CollinBeder** [Security-Onion-Solutions/securityonion#13394] [Unsupported Versions]
I created some test cases and now I need to delete them, is there a way to do this?

**Comment by dougburks** [comment]
You can set Status to `closed`:
![image](https://github.com/user-attachments/assets/318b5a8b-25b5-46e7-80ed-1422d79b4448)

By default, closed cases won't display on the main Cases screen since the default query is for open cases.
